src/color-chart/color-chart.js

```diff
@@ -478,7 +486,7 @@ const Self = class ColorChart extends ColorElement {
 					let range = this.xResolved?.refRange ?? this.xResolved?.range ?? [0, 100];
 					return range[0];
 				}
-				else if (!this.x || this.xMin === "auto") {
+				else if ((!this.x && isNaN(this.xMin)) || this.xMin === "auto") {
```

Comment:
```suggestion
				else if ((!this.x && force.min) || this.xMin === "auto") {
```
Basically, I'm trying to avoid having the logic for when we're forcing min and max spread over in multiple places.

Reply:
We don't have access to `force` here (it's inside props). Should we make it a prop or an instance property?